site/_posts/2017-10-23-survey-says.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Last Week in ManageIQ: Survey Says"
+author: jprause
+date: 2017-10-23
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hi all,...[John Prause][jprause] here. Last Week in ManageIQ, we reviewed the responses provided by our community users to a survey with five questions. The questions were as follows:
+  - How satisfied are you with the look and feel of the ManageIQ user interface?
+  - What are the challenges you face when getting started with ManageIQ?
+  - Do you attend the ManageIQ Sprint Reviews held every two weeks?
+  - Do you know how to contribute to the ManageIQ open source project?
+  - Is there anything else you’d like to share about ManageIQ?
+
+We had 38 people respond, and the comments and suggestions were very detailed and extremely helpful. We condensed the responses down to the Top 4 issues and our response:
+  - **Performance** - This is something our team is constantly working on, as all software developers should be. We see this as a journey that should never end, should be constantly be tested, and should be constantly reviewed. As such, Jason Frey (aka [Fryguy][fry]) recently posted a blog article regarding the [H-Release Rearchitecture][hrelease]. In this article you'll find that (as he says); _**The primary reason**_ for the rearchitecture _**was due to an increasing number of customer escalations**_ and _**problems of scalability and performance**_. I don't do his blog article any justice by showing snippets. Go read the article in its entirety and you'll see the commitment we have for you, our community users. Jason will also _be following up with some more blog posts detailing some of the many other rearchitecture decisions_.
```

Comment:
minor: Should be constantly ~~be~~ tested

It's a great post! I feel bad for nitpicking, but I'll merge right after this little little fix. Thanks @JPrause !